packages/preferences/src/node/json-preference-server.ts

```diff
@@ -86,8 +92,9 @@ export class JsonPreferenceServer implements PreferenceServer {
                 }
                 return this.fileSystem.resolveContent(uri).then(({ stat, content }) => {
                     const strippedContent = jsoncparser.stripComments(content);
-                    return JSON.parse(strippedContent)
-                })
+
+                    return JSON.parse(strippedContent);
```

Comment:
you are right, I just want to make sure that we can read at least some preferences if some comma is missing or trailing, not ignore all of them.